linkerd/stack/src/fallback.rs

```diff
@@ -1,11 +1,14 @@
 //! A middleware that may retry a request in a fallback service.
+
+use super::NewService;
 use futures::TryFuture;
 use linkerd2_error::Error;
 use pin_project::pin_project;
 use std::future::Future;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 use tower::util::{Either, Oneshot, ServiceExt};
+
 /// A Layer that augments the underlying service with a fallback service.
 ///
 /// If the future returned by the primary service fails with an error matching a
```

Comment:
it might be worth documenting the difference in behavior between the `NewService` and `MakeService` impls here?